src/App.tsx

```diff
@@ -1,8 +1,9 @@
 import React from "react";
-import { usersApi, User } from "./utils/fetcher";
+import { usersApi, User } from "utils/fetcher";
+import "./index.css";
```

Comment:
ini kenapa perlu import index.css lagi ya ?

Reply:
sama importnya kayaknya pake "@/index.css" aja 

```suggestion
import "@/index.css";
```

Reply:
> ini kenapa perlu import index.css lagi ya ?

jadi ga butuh file css jadinya ? `index.css` di rename dari `app.css`